tests/Makefile.am

```diff
@@ -10,6 +10,15 @@ CLEANFILES =
 TESTSUITE = $(srcdir)/rpmtests
 EXTRA_DIST += local.at $(TESTSUITE)
 
+AM_CPPFLAGS = -I$(top_srcdir)/include
+AM_CPPFLAGS += @WITH_OPENSSL_INCLUDE@
+AM_CPPFLAGS += @WITH_POPT_INCLUDE@
+AM_CPPFLAGS += -I$(top_srcdir)/misc
+AM_CPPFLAGS += -DLOCALEDIR="\"$(localedir)\""
+AM_CPPFLAGS += -DLIBRPMALIAS_FILENAME="\"rpmpopt-${VERSION}\""
```

Comment:
`-I$(top_srcdir)/include` is obviously necessary, but the rest doesn't belong at all AFAICS.